pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh

```diff
@@ -74,6 +77,19 @@ if [ "$(awk '$2 == "/" {print $3}' /proc/mounts)" == "tmpfs" ]; then
 				for DIR in ${DATADIRS}; do
 					DEST="/mnt/data$(dirname "${DIR}")"
 					mkdir -p "${DIR}" "${DEST}"
+					# Unmount filesystems (virtiofs, 9p, etc.) below "${DIR}", otherwise host mounts will be wiped out
```

Comment:
With this change the 5 lines above that start with `# Unmount all mount points under /tmp so we can move it to the data volume:` are redundant and can be removed.

Reply:
The previous lines will also break if the mount point includes spaces (which currently doesn't work anyways, but will be fixed by https://github.com/lima-vm/alpine-lima/pull/124).

Reply:
Removed the redundant unmount for /tmp